layouts/partials/widgets/experience.html

```diff
@@ -33,22 +34,34 @@ <h1>{{ with $page.Title }}{{ . | markdownify }}{{ end }}</h1>
         <div class="card">
           <div class="card-body">
             <h4 class="card-title exp-title text-muted mt-0 mb-1">{{.title | markdownify | emojify}}</h4>
-            <h4 class="card-title exp-company text-muted my-0">
-              {{- with .company_url}}<a href="{{.}}" target="_blank" rel="noopener">{{end}}{{.company | markdownify | emojify}}{{with .company_url}}</a>{{end -}}
-            </h4>
-            <div class="text-muted exp-meta">
-              {{ (time .date_start).Format ($page.Params.date_format | default "January 2006") }} –
-              {{ if .date_end}}
-                {{ (time .date_end).Format ($page.Params.date_format | default "January 2006") }}
-              {{else}}
-                {{ i18n "present" | default "Present" }}
-              {{end}}
-              {{with .location}}
-                <span class="middot-divider"></span>
-                <span>{{.}}</span>
-              {{end}}
+            {{- if .company_logo}}
+            <div class="d-flex">
```

Comment:
I had to add ` style="align-items: flex-start;"` to the `div` to prevent svg logos from stretching when I had text content which wrapped. Otherwise, I like and use this. Thank you.